base_layer/core/src/mempool/service/request.rs

```diff
@@ -27,6 +27,7 @@ use serde::{Deserialize, Serialize};
 pub type RequestKey = u64; // TODO: BaseNodeService and MempoolService uses RequestKey
 
 /// Generate a new random request key to uniquely identify a request and its corresponding responses.
+#[allow(dead_code)] // Squash the unused function warning
```

Comment:
It'd be better to post an issue to figure out if we can just remove this

Reply:
I could hide it behind the relevant feature flag actually though I was getting concerned about having too many feature flag conditionals

Reply:
Removed it rather, it is just for a warning that only occurs when building the wallet. I left a TODO explaining the issue.

Reply:
Removed it rather, it is just for a warning that only occurs when building the wallet. I left a TODO explaining the issue.

Reply:
Removed it rather, it is just for a warning that only occurs when building the wallet. I left a TODO explaining the issue.